src/common/lib/keylayers.c

```
@@ -0,0 +1,370 @@
+    .momentary_keys = {0},   // No MO keys held (supports layers 1 to MAX-1)
+    .oneshot_layer  = 0,     // No one-shot layer
+    .oneshot_active = false  // One-shot not active
+};
+
+/**
+ * @brief Reset layer state to base layer (layer 0)
+ *
+ * Clears all layer activations and returns to base layer.
+ * Used during initialization and error recovery.
+ */
+void keylayers_reset(void) {
```

Cppcheck (2.19.0)

[style] The function 'keylayers_reset' is never used.

(unusedFunction)